### PR TITLE
bump to v0.0.7 of osb-broker-lib

### PR DESCRIPTION
**Pull request by jboyd01**

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -35,5 +35,5 @@
 
 [[constraint]]
   name = "github.com/pmorie/osb-broker-lib"
-  version = "0.0.5"
+  version = "0.0.7"
```
Comment (LiliC) [inline Gopkg.toml]: @pmorie We really need to set up those e2e tests for this project, to make sure nothing breaks while bumping versions of the lib. ;)

Comment (pmorie) [reply Gopkg.toml]: Yes we do -- too bad there's no 'kube CI as a service :('

Comment (pmorie) [reply Gopkg.toml]: Now i'm getting some ideas, lol...

**Comment by pmorie** [comment]
Thanks @jboyd01

**Pull request merged**